demo/api/schema.gql

```diff
@@ -518,7 +518,7 @@ type Query {
   mainMenu(scope: PageTreeNodeScopeInput!): [PageTreeNode!]!
   brevoContact(id: Int!, scope: EmailCampaignContentScopeInput!): BrevoContact!
   brevoContacts(targetGroupId: ID, email: String, scope: EmailCampaignContentScopeInput!, offset: Int! = 0, limit: Int! = 25): PaginatedBrevoContacts!
-  manuallyAssignedBrevoContacts(offset: Int! = 0, limit: Int! = 25, targetGroupId: ID!, email: String): PaginatedBrevoContacts!
+  assignedBrevoContacts(offset: Int! = 0, limit: Int! = 25, targetGroupId: ID!, email: String, brevoId: Float): PaginatedBrevoContacts!
```

Comment:
brevo id should be an integer (not float), please check why the type is wrong here

Reply:
Thanks :) 
It's because of the wrong type in the args, as Raphi pointed out.